Assets/MixedRealityToolkit/_Core/Definitions/Devices/ControllerMappingLibrary.cs

```diff
@@ -224,10 +225,588 @@ public static class ControllerMappingLibrary
+                }
+
+                return wmrControllerLeftBlackScaled;
+            }
+        }
+
+        private static Texture2D wmrControllerRightWhite;
+
+        public static Texture2D WmrControllerRightWhite
+        {
+            get
+            {
+                if (wmrControllerRightWhite == null)
+                {
+                    wmrControllerRightWhite = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/MotionController_right_white.png", typeof(Texture2D));
+                }
+
+                return wmrControllerRightWhite;
+            }
+        }
+
+        private static Texture2D wmrControllerRightWhiteScaled;
+
+        public static Texture2D WmrControllerRightWhiteScaled
+        {
+            get
+            {
+                if (wmrControllerRightWhiteScaled == null)
+                {
+                    wmrControllerRightWhiteScaled = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/MotionController_right_white_scaled.png", typeof(Texture2D));
+                }
+
+                return wmrControllerRightWhiteScaled;
+            }
+        }
+
+        private static Texture2D wmrControllerRightBlack;
+
+        public static Texture2D WmrControllerRightBlack
+        {
+            get
+            {
+                if (wmrControllerRightBlack == null)
+                {
+                    wmrControllerRightBlack = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/MotionController_right_black.png", typeof(Texture2D));
+                }
+
+                return wmrControllerRightBlack;
+            }
+        }
+
+        private static Texture2D wmrControllerRightBlackScaled;
+
+        public static Texture2D WmrControllerRightBlackScaled
+        {
+            get
+            {
+                if (wmrControllerRightBlackScaled == null)
+                {
+                    wmrControllerRightBlackScaled = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/MotionController_right_black_scaled.png", typeof(Texture2D));
+                }
 
-        #endregion
+                return wmrControllerRightBlackScaled;
+            }
+        }
 
-        #endregion Controller axis mapping configuration
+        private static Texture2D touchControllerLeftWhite;
+
+        public static Texture2D TouchControllerLeftWhite
+        {
+            get
+            {
+                if (touchControllerLeftWhite == null)
+                {
+                    touchControllerLeftWhite = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/OculusControllersTouch_left_white.png", typeof(Texture2D));
```

Comment:
I would comment / remove this code (entire touch controller block) while awaiting guidance on images

Reply:
As above